pkg/specgen/podspecgen.go

```diff
@@ -57,6 +57,8 @@ type PodBasicConfig struct {
 	// (e.g. `podman generate systemd --new`).
 	// Optional.
 	PodCreateCommand []string `json:"pod_create_command,omitempty"`
+	// Pid sets the process id namespace of the pod
```

Comment:
Add a line underneath - "Optional (defaults to private if unset)". And this should mention that it sets the PID namespace of the infra container, that will then be shared with the entire pod if PID namespace sharing is enabled.